docs/UserGuide.md

```diff
@@ -45,142 +38,291 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 
 Format: `help`
 
-> Help is also shown if you enter an incorrect command e.g. `abcd`
+> * Help is also shown if you enter an incorrect command e.g. `abcd`
 
-### 2.2. Adding a person: `add`
+### 2.2. Adding a task: `add`
 
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Adds a task to DoOrDie<br />
+Format: `add TASKNAME [label LABEL] ([(by|on) DEADLINE] | [from START_DATE to END_DATE]) [repeat (hourly|daily|weekly|monthly|yearly)]`
 
-> Persons can have any number of tags (including 0)
+> * Tasks can have a deadline, or can do without one as well.
+>   * Tasks added without specifying a deadline will be displayed under "No Deadline".
+>   * Date formats can be flexible. The application is able to parse commonly-used human-readable date formats.
+>     * e.g. `Monday`, `next wed`, `tomorrow`, `5 days after`, `4 Apr` etc.
+> * Dates can include time as well.
+>   * If only time is specified, it will default to today's date.
+>   * If time is not specified, it will default to the current time of the particular date.
+>   * Time formats are flexible as well. The application supports 24 hour format and AM/PM format.
+>     * e.g. `Monday 3pm`, `today 1930`, `5:30pm`, `10.00 am`
+> * Recurring task will have the same deadline if `daily/weekly/monthly/yearly` is used
```

Comment:
E.g. add task by 2300 Monday repeat daily will create task that every 2300 daily. Should we clarify on this?